Header header

#Request types
	int8 CMD_OPEN = 0
	int8 CMD_CLOSE = 1
	int8 LEFT = 3
	int8 RIGHT = 4
	int8 BOTH = 5
	
int8 cmd
int8 limb
